scripts/clear-all-fault-leds.sh

```diff
@@ -4,6 +4,14 @@
 # /xyz/openbmc_project/State/Decorator/OperationalStatusManager
 # to true or false.
 
+# Skip running this script during a warm reboot(graceful/force).
```

Comment:
This comment needs correction. Skip if the chassis is powered ON.

Reply:
done